src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParameters.kt

```diff
@@ -59,7 +59,7 @@ class QuickAccessParameters(_project: Project) {
     private var stoppingConditionToolTip =
         JBLabel("Default: 60 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
     private var populationLimitToolTip =
-        JBLabel("Default: 60 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
+        JBLabel("Default: 50 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
```

Comment:
Why did you change this?

Reply:
https://github.com/EvoSuite/evosuite/blob/master/client/src/main/java/org/evosuite/Properties.java

Reply:
Ah, I see. Thanks for clearing this up.